blog-post-review.md

```diff
@@ -0,0 +1,261 @@
+
+After you cloned the repo and positioned yourself in it, now we can start the `mage-cugraph` image:
+
+```
+docker run -it -p 7687:7687 -p 7444:7444 --volume ./data/facebook_clean_data/:/samples mage-cugraph
+```
+
+
+### Jupyter notebook
+After you have Memgraph running, you should install **Jupyter**. For this tutorial, we used **jupyter-lab** and you can install it with the following command:
+```
+pip install jupyterlab
+```
+
+Now you can go and download **[jupyter-notebook](https://github.com/memgraph/jupyter-memgraph-tutorials/tree/main/cugraph-analytics)** we use in our tutorial. Note, in a repository folder `cugraph-analytics/data` there are `CSV` files containing the full dataset which we will use later in the tutorial.
+
+
+### GQLAlchemy installation
+We will be using the **[GQLAlchemy](https://memgraph.com/docs/gqlalchemy/)** object graph mapper (OGM) to connect to Memgraph and execute **Cypher** queries from **Python**. You can think of Cypher as SQL for graph databases. It contains many of the same language constructs like `CREATE`, `UPDATE`, `DELETE`... and it's used to query the database. GQLAlchemy serves as a Python driver/client for Memgraph. Go to our [installation](https://memgraph.com/docs/gqlalchemy/installation) page to check how more options on how to install it. If you wish, you can install it with `pip`, but be sure also to have `CMake` installed. After you have `CMake` ready, you can install `GQLAlchemy` with the following command:
+```
+pip install gqlalchemy
+```
+
+### Memgraph Lab installation
+Last piece of tech you will use is **[Memgraph Lab]()**. You will use it to connect to **Memgraph** and create visualizations. To check how to install **Memgraph Lab** and connect to **Memgraph**, you can go to one of the following pages for installation instructions for **Memgraph Lab as desktop application** for [Windows](https://memgraph.com/docs/memgraph-lab/installation/windows#step-2---installing-and-setting-up-memgraph-lab), [Mac](https://memgraph.com/docs/memgraph-lab/installation/macos#step-2---installing-and-setting-up-memgraph-lab) or [Linux](https://memgraph.com/docs/memgraph-lab/installation/linux#step-2---installing-and-setting-up-memgraph-lab). After you have installed **Memgraph Lab**, you can check instructions how to **[connect to Memgraph](https://memgraph.com/docs/memgraph-lab/connect-to-memgraph#connecting-to-memgraph)** with Memgraph Lab. 
```

Comment:
The last piece of tech you will need is Memgraph Lab.

...create visualizations OF WHAT? I feel like something is missing... create data visualizations?

[After that, it sounds like you need to hit a certain word count mark so you are writing as many words as possible to say something :) ]

Check out how to install Memgraph Lab as a desktop application for W, M or L and then connect to Memgraph database. 

[That's all you need to tell them :) ]

With the Memgraph Lab installed and connected, we are ready to connect to Memgraph with GQLAlchemy, import our large dataset and run graph analytics using Python.